src/Service/Storage/Ef/DbGlobalContext.cs

```diff
@@ -68,6 +69,16 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                 .HasDefaultValue(true);
         });
 
+        modelBuilder.Entity<Authenticator>(b =>
+        {
+            b.HasKey(x => new { x.Tenant, x.AaGuid });
+
+            b.HasOne(x => x.AppFeature)
+                .WithMany(x => x.Authenticators)
+                .HasForeignKey(x => x.Tenant)
+                .IsRequired();
+        });
```

Comment:
issue: This DBSet is missing the query filtering in `EFTenantContext` so any queries would impact all apps, not just the specific tenant.